pdfminer/arcfour.py

```diff
@@ -5,9 +5,12 @@
 """
 
 
+from typing import Sequence
+
+
 class Arcfour:
 
-    def __init__(self, key):
+    def __init__(self, key: Sequence[int]):
```

Comment:
Sometimes you explicitly annotate the `None` return type but here you don't. Is that arbitrary or by choice?

Reply:
I'll go back and fix these. I learned only by experience that mypy insists on -> None only when a constructor has no arguments, but for completeness they should all have it.